SerialPrograms/Source/CommonFramework/ImageMatch/WaterfillTemplateMatcher.cpp

```diff
@@ -140,13 +131,50 @@ double WaterfillTemplateMatcher::rmsd_original(const ImageViewRGB32& original_im
     }
 
     double rmsd = this->rmsd(extract_box_reference(original_image, object));
-    if (PreloadSettings::debug().IMAGE_TEMPLATE_MATCHING){
-        cout << "Passed aspect and area ratio check, rmsd = " << rmsd << endl;
-    }
-   
 
     return rmsd;
 }
+void WaterfillTemplateMatcher::check_aspect_ratio_debug(size_t candidate_width, size_t candidate_height) const {

```

Comment:
Not a real fan of duplicating the logic between the main code and the debugging code here. Because it creates an opportunity to for them to accidentally get out-of-sync which would make the debug versions behave differently from the main ones.